code/src/App.js

```diff
@@ -1,9 +1,40 @@
-import React from 'react'
+import React, { useEffect, useState } from 'react'
+
+import { HappyThoughts } from './Components/HappyThoughts'
+import { HappyForm } from './Components/HappyForm'
+
+const url = "https://happy-thoughts-technigo.herokuapp.com/thoughts"
 
 export const App = () => {
+  const [thoughts, setThoughts] = useState([])
+  const [postedMessage, setPostedMessage] = useState("")
+
+  useEffect (() => {
+    fetch (url)
+    .then(res => res.json())
+    .then(json => setThoughts(json))
+  }, [postedMessage])
+
+  const handleFormSubmit = message => {
+    fetch (url, { method:"POST", body:JSON.stringify({message}), headers:{"Content-Type":"application/json"}})
+    .then(() => setPostedMessage(message))
+    .catch(err => console.log("error:", err))
+  }
+
```

Comment:
maybe you could use some indentation here to make it clearer and more easy to read? like:
```
fetch(url, {
   method: 'post',
   body: JSON.stringify({message}
   headers:{"Content-Type":"application/json"}
}).then…

```